src/test/regress/expected/shard_rebalancer.out

```diff
@@ -438,6 +438,9 @@ CREATE VIEW replication_test_table_placements_per_node AS
     AND shardstate != 4
     GROUP BY nodename, nodeport
     ORDER BY nodename, nodeport;
+WARNING:  "view replication_test_table_placements_per_node" has dependency to "table replication_test_table" that is not in Citus' metadata
```

Comment:
ah, we have this warning because table is created with master_create_distributed_table and the table is not in pg_dist_object?

Reply:
Yes